src/apps/relay/userdb.c

```diff
@@ -711,6 +727,18 @@ void release_allocation_quota(uint8_t *user, int oauth, uint8_t *realm) {
     ur_string_map_unlock(rp->status.alloc_counters);
     free(username);
   }
+
+  int log_level = TURN_LOG_LEVEL_DEBUG;
```

Comment:
Why changing loglevel? Is it debugging code?

Reply:
@eakraly As a reminder your previous comment was:
"This can be INFO when the server is in drain mode - I think it will be useful to see in logs that the draining is happening...
But not while in regular operation. We already have too much logging."

So I made that change to log at DEBUG normally and at INFO when we are draining.  :)

Reply:
To be clear we are not changing the logging level, we are setting the level at which to send to the TURN_LOG_FUNC.